minard/views.py

```diff
@@ -3,6 +3,7 @@
 from flask import render_template, jsonify, request, redirect, url_for
 from itertools import product
 import time
+from pytz import timezone
```

Comment:
I can probably figure out a way to make this work without pytz. For the record though, the issue (partially) was that the DB stores the timestamp info without any associate timezone info, a reasonable thing to do since the assumption that the timestamp is in Sudbury time is pretty safe. However it seems the way python handles timestamps that don't have timezone info is pretty demented. It automatically assumes the time is UTC and using pytz was the only way I could figure out to convince python otherwise.

However, since I already do some string editing of the timestamp [here ](https://github.com/snoplus/minard/pull/17/files/6f5eeb5fb861df9eb89ca7e848adfe12ac227653#diff-0cec7c20af1ce13808baeff5630b600bR537)maybe I can just make that a little more extreme and fix the whole thing.